raw_description/urdf/vacuum_gripper/gripper.urdf.xacro

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:include filename="$(find raw_description)/urdf/vacuum_gripper/gripper.gazebo.xacro" />
+
+  <xacro:property name="gripper_length" value="0.14"/>
+  <xacro:property name="gripper_width" value="0.2"/>
+  <xacro:property name="gripper_height" value="0.14"/>
+  <xacro:property name="gripper_body_height" value="0.14"/>
+  <xacro:property name="radius" value="0.005"/>
+
+  <xacro:macro name="vacuum_gripper" params="name parent *origin">
+
+    <!-- joint between arm_ee_link and raw_boxgripper_link -->
+    <joint name="${name}_base_joint" type="fixed" >
+        <insert_block name="origin" />
+        <parent link="${parent}" />
+        <child link="gripper_frame" />
+    </joint>
+
+    <link name="gripper_frame"/>
+
+    <joint name="gripper_to_palm" type="fixed">
+        <parent link="gripper_frame"/>
+        <child link="gripper_body"/>
+        <origin xyz="0.08015 0.0475 0.0" rpy="1.57079632679 0 3.1415"/>
+    </joint>
+
+    <link name="gripper_body">
+        <visual>
+            <!--origin xyz="0 0.036 -0.0473" rpy="0 0 0"/-->
+              <!--geometry>
+                <box size="${gripper_length} ${gripper_width} ${gripper_height}"/>
+              </geometry-->
+            <origin xyz="0 0 -0.048" rpy="0 0 0"/>
+            <geometry>
+                <mesh filename="package://raw_description/meshes/vacuum_gripper/segmentgreifer_2.stl" />
+            </geometry>
+            <material name="silver">
+                <color rgba="0.8 0.8 0.8 1"/>
+            </material>
+        </visual>
+        <collision>
+            <origin xyz="0 0.03 -0.05"/>
+            <geometry>
+                <box size="${gripper_length} ${gripper_width} ${gripper_height}"/>
+            </geometry>
```

Comment:
use the [box_geometry mesh](https://github.com/ipa320/cob_common/blob/indigo_dev/cob_description/urdf/common.xacro#L19-L23)